packages/web/e2e/signUp.test.ts

```diff
@@ -0,0 +1,277 @@
+import path from 'path'
+
+import { Page, expect, devices } from '@playwright/test'
+
+import { test } from './test'
+import { resetAuthState, generateTestUser } from './utils'
+
+type User = {
+  email: string
+}
+
+async function expectSignUpPage(page: Page) {
+  await page.waitForURL(/signup/i)
+  await expect(
+    page.getByRole('heading', { name: /sign up for audius/i, level: 1 })
+  ).toBeVisible()
+}
+
+async function assertOnCreatePasswordPage(user: User, page: Page) {
+  await expect(
+    page.getByRole('heading', { name: /create your password/i })
+  ).toBeVisible()
+
+  await expect(
+    page.getByText(/Create a password that’s secure and easy to remember!/i)
+  ).toBeVisible()
+
+  await expect(page.getByText(/your email/i)).toBeVisible()
+  await expect(page.getByText(user.email)).toBeVisible()
+}
+
+async function testSignUp({
+  isMobile = false,
+  testUser = generateTestUser(),
+  signUpUrl = 'signup',
+  page
+}: {
+  isMobile?: boolean
+  testUser?: ReturnType<typeof generateTestUser>
+  signUpUrl?: string
+  page: Page
+}) {
+  const { email, password, handle, name } = testUser
+
+  // --> Load signup route
+  await page.goto(signUpUrl)
+  await expect(
+    page.getByRole('heading', { name: /sign up for audius/i, level: 1 })
+  ).toBeVisible({ timeout: 45000 })
+
+  await page.getByRole('button', { name: /sign up free/i }).click()
+
+  // --> Email Page
+  await page.getByRole('textbox', { name: /email/i }).fill(email)
+  await page.getByRole('button', { name: /sign up free/i }).click()
+
```

Comment:
I debated about making POMs for these sign up pages but ultimately didn't since it would just be a bunch of mini POMs and we won't be reusing them anywhere (since this is the only test that will target sign up).
Still considering it though so I could probably be persuaded

Reply:
One thing that might be reusable is the enter credentials page and OTP page (for change email/change password) but I think we cross that bridge when we get there